### ISSUE TITLE
 0. Crash: java.lang.IllegalArgumentException: You cannot start a load for a destroyed activity

### ISSUE THREAD

**Issue by noinlijin** [intercom/intercom-android#397]
Please use the following template to submit your issue. Following this template will allow us to quickly investigate and help you with your issue. Please be aware that issues which do not conform to this template may be closed.

For feature requests please contact us at team@intercom.io

## Version info
  - Intercom for Android version:3.2.10
  - Android OS version: 4.4.4 1920x1200 armeabi-v7a
  - Android Device:Lenovo TAB 2 A10-70LC

## Steps to reproduce

![image](https://user-images.githubusercontent.com/3073370/29060429-564edc36-7c4c-11e7-8822-5225f687b3a2.png)

## Logs
```
java.lang.IllegalArgumentException: You cannot start a load for a destroyed activity
at a.adh.b(Unknown Source)
at a.adh.a(Unknown Source)
at a.adh.a(Unknown Source)
at a.adh.a(Unknown Source)
at a.ue.b(Unknown Source)
at io.intercom.android.sdk.conversation.composer.galleryinput.GalleryImageLoader.loadImageIntoView(Unknown Source)
at com.intercom.input.gallery.adapter.c.a(Unknown Source)
at com.intercom.input.gallery.adapter.c.onBindViewHolder(Unknown Source)
at android.support.v7.widget.ck.onBindViewHolder(Unknown Source)
at android.support.v7.widget.ck.bindViewHolder(Unknown Source)
at android.support.v7.widget.de.a(Unknown Source)
at android.support.v7.widget.de.a(Unknown Source)
at android.support.v7.widget.de.a(Unknown Source)
at android.support.v7.widget.de.c(Unknown Source)
at android.support.v7.widget.bw.a(Unknown Source)
at android.support.v7.widget.GridLayoutManager.a(Unknown Source)
at android.support.v7.widget.LinearLayoutManager.a(Unknown Source)
at android.support.v7.widget.LinearLayoutManager.c(Unknown Source)
at android.support.v7.widget.GridLayoutManager.c(Unknown Source)
at android.support.v7.widget.RecyclerView.N(Unknown Source)
at android.support.v7.widget.RecyclerView.p(Unknown Source)
at android.support.v7.widget.RecyclerView.onLayout(Unknown Source)
at android.view.View.layout(View.java:15167)
at android.view.ViewGroup.layout(ViewGroup.java:4894)
at android.widget.FrameLayout.layoutChildren(FrameLayout.java:500)
at android.widget.FrameLayout.onLayout(FrameLayout.java:435)
at android.view.View.layout(View.java:15167)
at android.view.ViewGroup.layout(ViewGroup.java:4894)
at android.widget.FrameLayout.layoutChildren(FrameLayout.java:500)
at android.widget.FrameLayout.onLayout(FrameLayout.java:435)
at android.view.View.layout(View.java:15167)
at android.view.ViewGroup.layout(ViewGroup.java:4894)
at android.support.design.widget.CoordinatorLayout.c(Unknown Source)
at android.support.design.widget.CoordinatorLayout.a(Unknown Source)
at android.support.design.widget.CoordinatorLayout.onLayout(Unknown Source)
at android.view.View.layout(View.java:15167)
at android.view.ViewGroup.layout(ViewGroup.java:4894)
at android.support.design.widget.CoordinatorLayout.c(Unknown Source)
at android.support.design.widget.CoordinatorLayout.a(Unknown Source)
at android.support.design.widget.CoordinatorLayout.onLayout(Unknown Source)
at android.view.View.layout(View.java:15167)
at android.view.ViewGroup.layout(ViewGroup.java:4894)
at android.support.design.widget.CoordinatorLayout.c(Unknown Source)
at android.support.design.widget.CoordinatorLayout.a(Unknown Source)
at android.support.design.widget.CoordinatorLayout.onLayout(Unknown Source)
at android.view.View.layout(View.java:15167)
at android.view.ViewGroup.layout(ViewGroup.java:4894)
at android.widget.FrameLayout.layoutChildren(FrameLayout.java:500)
at android.widget.FrameLayout.onLayout(FrameLayout.java:435)
at android.view.View.layout(View.java:15167)
at android.view.ViewGroup.layout(ViewGroup.java:4894)
at android.widget.FrameLayout.layoutChildren(FrameLayout.java:500)
at android.widget.FrameLayout.onLayout(FrameLayout.java:435)
at android.view.View.layout(View.java:15167)
at android.view.ViewGroup.layout(ViewGroup.java:4894)
at android.widget.FrameLayout.layoutChildren(FrameLayout.java:500)
at android.widget.FrameLayout.onLayout(FrameLayout.java:435)
at android.view.View.layout(View.java:15167)
at android.view.ViewGroup.layout(ViewGroup.java:4894)
at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1883)
at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1737)
at android.widget.LinearLayout.onLayout(LinearLayout.java:1646)
at android.view.View.layout(View.java:15167)
at android.view.ViewGroup.layout(ViewGroup.java:4894)
at android.widget.FrameLayout.layoutChildren(FrameLayout.java:500)
at android.widget.FrameLayout.onLayout(FrameLayout.java:435)
at android.view.View.layout(View.java:15167)
at android.view.ViewGroup.layout(ViewGroup.java:4894)
at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:2510)
at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2203)
at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1262)
at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:6400)
at android.view.Choreographer$CallbackRecord.run(Choreographer.java:788)
at android.view.Choreographer.doCallbacks(Choreographer.java:591)
at android.view.Choreographer.doFrame(Choreographer.java:560)
at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:774)
at android.os.Handler.handleCallback(Handler.java:808)
at android.os.Handler.dispatchMessage(Handler.java:103)
at android.os.Looper.loop(Looper.java:193)
at android.app.ActivityThread.main(ActivityThread.java:5322)
at java.lang.reflect.Method.invokeNative(Native Method)
at java.lang.reflect.Method.invoke(Method.java:515)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:829)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:645)
at dalvik.system.NativeStart.main(Native Method)
```

**Comment by ronocod** [comment]
How often are you seeing this crash?

**Comment by noinlijin** [comment]
I use aliyun free test 30 devices, [here](http://mqc.aliyun.com/)

**Comment by ronocod** [comment]
Fixed in 4.0.3
